src/js/common/schemaform/ArrayField.jsx

```diff
@@ -0,0 +1,324 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import Scroll from 'react-scroll';
+import { focusElement } from '../utils/helpers';
+
+import {
+  retrieveSchema,
+  toIdSchema,
+  getDefaultFormState,
+  deepEquals
+} from 'react-jsonschema-form/lib/utils';
+
+import { errorSchemaIsValid } from './helpers';
+
+const Element = Scroll.Element;
+const scroller = Scroll.scroller;
+
+const scrollToFirstError = () => {
```

Comment:
We seem to have this basic function in a lot of places. Would it be useful to export it into helpers?